components/kyma-environment-broker/internal/broker/instance_update.go

```diff
@@ -112,3 +124,17 @@ func (b *UpdateEndpoint) Update(ctx context.Context, instanceID string, details
 		OperationData: "",
 	}, nil
 }
+
+func (b *UpdateEndpoint) exctractActiveValue(id string, provisioning internal.ProvisioningOperation) (*bool, error) {
+	deprovisioning, dErr := b.operationStorage.GetDeprovisioningOperationByInstanceID(id)
+	if dErr != nil && !dberr.IsNotFound(dErr) {
```

Comment:
nit:
i don't see a reason for `dErr` naming - maybe let's stick with the native Go naming `err`